### DISCUSSION TITLE
Can not find boot partition on Polarfire Soc "MPFS250-Video-Kit" [Flash using eMMC]

### DISCUSSION THREAD

**Discussion by hiteshmathu** [polarfire-soc/polarfire-soc-documentation#415] [Applications and demos]
1) when i try Flash using eMMC, I m not able to find the boot image virtual drive instead show only one drive i.e E:/. so, how can create new partition and add boot image on it. 

![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/15924135/dc552856-3ae5-4ddf-9325-5f9e08229c15)

 
![image](https://github.com/polarfire-soc/polarfire-soc-documentation/assets/15924135/793cbd90-3247-451c-b83b-a940b9aac167)

2) can I flash non-secure boot Linux image?

Please help me on this.... TIA

**Comment by sc-mchp** [comment]
It should work if your able to detect the portion also.
In screen short your using ICICLE kit wic image plz use mpfs-vido-kit wic image and job from the latest release.
